src/help.ts

```diff
@@ -2,7 +2,7 @@
 
 import * as config from "./config"
 
-/** Created the element that should contain keybinding information */
+/** Creates the element that should contain keybinding information */
```

Comment:
If we're going to be pedantic, we probably want the imperative voice here - "Create".